README.md

```diff
@@ -1079,8 +1079,12 @@ Flags:
   - :nerd_face: `--format=wide`: Alias of `--format=table`
   - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
 - :nerd_face: `--size`: Display the disk usage of volumes.
-
-Unimplemented `docker volume ls` flags: `--filter`
+- :whale: `-f, --filter`: Filter volumes based on given conditions.
+    - :whale: `--filter label=<key>=<value>`: Matches volumes by label on both
+      `key` and `value`. If `value` is left empty, matches all volumes with `key`
+      regardless of its value
+    - :whale: `--filter name=<value>`: Matches all volumes with a name containing
+      the `value` string
```

Comment:
maybe we can and a group of operator in argument

1. size>=:int
2. size>int
3. size<=int
4. size<int

Reply:
More description plz. Don't understand.

Reply:
In volumes data structure 

```go
type Volume struct {
	Name       string             `json:"Name"`
	Mountpoint string             `json:"Mountpoint"`
	Labels     *map[string]string `json:"Labels,omitempty"`
	Size       int64              `json:"Size,omitempty"`
}
```

We record the data size for the volume. So maybe it's better to allow people filtering the volume by data size range